source/coll_common.f90

```diff
@@ -122,13 +122,12 @@ module coll_common
 
 contains
 
-subroutine coll_expandArrays(npart_new, nblz_new)
```

Comment:
I guess that here `nblz_new` was added to the interface just for the sake of completeness, such that in case one need to add stuff depending on the new number of lattice structures, the interface is already general enough

Reply:
It's probably no longer needed after we killed a bunch of collimation arrays.